### PR TITLE
Mention Troubleshooting guide at the top of settingUp.md #5113

### LINKED ISSUE #5113: Mention Troubleshooting guide at the top of settingUp.md
We can add the following at the top of the `settingUp.md`

If you encounter any problems during the setting up process, please refer our [troubleshooting guide]() before posting a help request in our [issue tracker]().

### PR DESCRIPTION
**Pull request by onkarshedge**
Fixes #5113

### CODE REVIEW

File: devdocs/settingUp.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #Setting Up the Developer Environment
+If you encounter any problems during the setting up process, please refer our [troubleshooting guide](https://docs.google.com/document/d/1_p7WOGryOStPfTGA_ZifE1kVlskb1zfd3HZwc4lE4QQ/pub?embedded=true) before posting a help request in our [issue tracker](https://github.com/TEAMMATES/teammates/issues).
+
```
Comment (damithc) [inline devdocs/settingUp.md]: Perhaps add a `>` at the front to show it as a block quote?

File: devdocs/settingUp.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 #Setting Up the Developer Environment
+>If you encounter any problems during the setting up process, please refer our [troubleshooting guide](https://docs.google.com/document/d/1_p7WOGryOStPfTGA_ZifE1kVlskb1zfd3HZwc4lE4QQ/pub?embedded=true) before posting a help request in our [issue tracker](https://github.com/TEAMMATES/teammates/issues).
```
Comment (wkurniawan07) [inline devdocs/settingUp.md]: ... refer **to** our ...

**Comment by wkurniawan07** [comment]
One minor improvement, other than that, good to go.

**Comment by wkurniawan07** [comment]
@onkarshedge next time, you should let the reviewer know when your work is ready for review again

**Comment by damithc** [comment]
@onkarshedge the branch name has an extra dash, but we'll overlook it this time.

**Comment by onkarshedge** [comment]
Sorry, won't happen again. @wkurniawan07  yes.

**Pull request merged**